### DISCUSSION TITLE
Device Group - What happens when the user is deleted.

### DISCUSSION THREAD

**Discussion by timcanty** [Ylianst/MeshCentral#6034] [General]
Hi All,

Daft question, when a user has setup loads of device groups and these groups are shared with other users.

If that user is then deleted, do the device groups stay, or do they get deleted with the user?

Don't think i have come accross the synario before, and before i cause myself a headache thought would ask the question?

Kind Regards

**Comment by si458** [comment]
i cant fully remember but
i think if you delete the user and a user has their own groups, those groups get deleted with it
and if you delete a user that doesnt have their own groups but groups shared to it, only the user is deleted as they dont have control over the group as the group is shared to them

**Reply by timcanty** [reply]
There isn't a way to move a group to a different owner is there?
